packages/dart_frog_cli/lib/src/dev_server_runner/dev_server_runner.dart

```diff
@@ -95,6 +98,9 @@ class DevServerRunner {
   /// [Logger] instance used to wrap stdout.
   final Logger logger;
 
+  /// Which host the server should start on.
```

Comment:
**Nit:**

In the command option help we are using "Which host the server should bind to.", whereas here we are using "Which host the server should start on.". I think we should ensure they are consistent.

I prefer (personal preference) the former one, since the shelf package also uses "bind" as the name of the serve constructor.

Reply:
Check [commit](https://github.com/VeryGoodOpenSource/dart_frog/pull/924/commits/76d30d627c4a31612cb7ed27d99e6fc6eefdd8e1)